### DISCUSSION TITLE
Github education pack shows as "Approved", but still not reflecting in account.

### DISCUSSION THREAD

**Discussion by itsKarad** [community/community#67909] [GitHub Education]
### Hi everyone,

"Congrats! Your request for @ myusername was approved and your benefits will be available soon.". It's been 2 days. How long does this usually take?

**Comment by mantasu** [comment]
Hey, I had the same issue. Quoting from [support page](https://support.github.com/request/education?tags=docs-generic):

> Applications may take several weeks to be approved. We’re working as fast as we can but the review process can take a while especially during times where we receive high volumes of applications, such as the start of a new semester.

If it says approved and it is green that means you still have to wait for a bit before you get access to the benefits. Once the "Approved" tag is purple, it should be ready:

![Screenshot 2023-09-24 072637](https://github.com/community/community/assets/56790921/30412a58-0e47-4b3c-8f87-fe9c85ec62bb)

Otherwise, after a few of weeks if you still don't have access, you may want to open a [ticket](https://support.github.com/request/education?tags=docs-generic).

**Reply by mantasu** [reply]
If you haven't received the benefits yet, I saw that staff is suggesting reapplying in some cases, for example [here](https://github.com/orgs/community/discussions/69133#discussioncomment-7185814), so worst case I would suggest to just re-raise the ticket and reapply :)
